lib/contracts.js

```diff
@@ -18,9 +18,13 @@ export const runUpdateContractsLoop = async ({ provider, abi, contracts }) => {
       if (err.name === 'AbortError') return
       throw err
     }
-    const newContracts = await getContractsWithRetry({ provider, abi })
-    contracts.splice(0)
-    contracts.push(...newContracts)
+    try {
+      const newContracts = await getContractsWithRetry({ provider, abi })
+      contracts.splice(0)
+      contracts.push(...newContracts)
+    } catch (err) {
+      console.error('Failed to update the list of contract addresses. Will retry later.', err)
```

Comment:
What do you think about creating an activity entry here? Say this always fails, then the user should know.

Reply:
See [14f93ab](https://github.com/filecoin-station/core/pull/474/commits/14f93ab87dc1c84088218ac7fabe5805b10429ae)